contracts/factories/WrappedExternalBribeFactory.sol

```diff
@@ -4,14 +4,11 @@ pragma solidity 0.8.13;
 import {WrappedExternalBribe} from 'contracts/WrappedExternalBribe.sol';
 
 contract WrappedExternalBribeFactory {
-    address public immutable voter;
+    bool internal _initialized;
+    address public voter;
     mapping(address => address) public oldBribeToNew;
     address public last_bribe;
 
-    constructor(address _voter) {
```

Comment:
[Question] why do we remove setting voter in constructor?

Reply:
@t0rbik

Reply:
> [Question] why do we remove setting voter in constructor?
> 
> imo, if there is no specific reasons, please undo the changes in this contract

There's a particular reason.
In order for voter to automate creation of wrapped external bribe, we added a wrapped external bribe factory to voter constructor. That's being said, we need to deploy wrapped external bribe factory before voter, thus voter can't be passed to constuctor. After contracts are deployed, idea is to setVoter in init script.